packages/rum-recorder/src/domain/privacy.ts

```diff
@@ -0,0 +1,31 @@
+const PRIVACY_ATTR_NAME = 'data-dd-privacy'
+const PRIVACY_ATTR_VALUE_HIDDEN = 'hidden'
+
+const PRIVACY_CLASS_HIDDEN = 'dd-privacy-hidden'
+
+// Returns true if the given DOM node should be hidden. Ancestors
+// are not checked.
+export function nodeIsHidden(node: Node): boolean {
```

Comment:
naming: I found `nodeIsHidden` a bit confusing since it is not hidden yet, maybe `nodeShouldBeHidden` or `nodeIsMarkedHidden` instead?
same logic for the other function

Reply:
I see your point. I don't have strong feeling, so I'll go with `should`